apps/emqx_utils/test/emqx_utils_binary_tests.erl

```diff
@@ -0,0 +1,217 @@
+%%--------------------------------------------------------------------
+%% Original file taken from https://github.com/arcusfelis/binary2
+%% Copyright (c) 2016 Michael Uvarov
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+-module(emqx_utils_binary_tests).
+
+-import(emqx_utils_binary, [
+    trim/1,
+    ltrim/1,
+    rtrim/1,
+    trim/2,
+    ltrim/2,
+    rtrim/2,
+    reverse/1,
+    inverse/1,
+    join/2,
+    suffix/2,
+    prefix/2,
+    duplicate/2,
+    union/2,
+    intersection/2,
+    subtract/2,
+    optimize_patterns/1
+]).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-ifdef(TEST).
```

Comment:
I think this is not necessary, as this is in the test directory path already.

Reply:
I know, “as found”. Will remove, doesn’t make much sense to me either.